### DISCUSSION TITLE
Part of Main menu dissapeared after update

### DISCUSSION THREAD

**Discussion by patrikzudel** [johnlindquist/kit#1363] [Q&A]
I've updated to 1.98.9 today and I've noticed that this part of the menu is gone, is this intentional or a bug?

![blablal2222ba](https://github.com/johnlindquist/kit/assets/34461296/f5e615b0-295e-43d0-8e24-3be92f535e86)

Thanks!

**Comment by johnlindquist** [comment]
Intentional.

In 2.0, we're using keywords to access the menus that were hidden in the tabs

**Comment by patrikzudel** [comment]
Frankly, I see this as a huge step backwards in user friendliness. Using Keywords would be a nice way to jump in between these sub-menus (for keyboard only use) but to delete them completely has big implications for new users. If I hadn't seen that there is a Guide or API in the previous version I wouldn't even know that I can search for them. The sub-menus gave me **instant** key usage information without me having to intentionally search for them, which is huge from a UX standpoint.

Also, the UI now seems much more chaotic, we have a **huge** vertical wall of text which, without horizontal categorization just seems like a soup of text. Very easy to miss singular options like the "Browse Kit API" etc. I also dislike that the community tab is now combined with my scripts, it made way more sense before. 

Please consider adding it back, or giving users the option to toggle it (making the way it was before the default).

**Reply by johnlindquist** [reply]
"Huge" wall of text? It should be the same height as before. Did something happen to resize the main menu? You can manually resist it or reset it with cmd+0.

User feedback was that nobody used the tabs and they'd rather discover through text search. I can make it an option to restore the tabs if that's your preference.

**Reply by patrikzudel** [reply]
The window is the same height, but without horizontal tabs (sub-menus) there only is a huge amount of vertical distance the user has to traverse using scrolling, that's what I meant by wall of text. I apologize, I could have worded it more precisely.

I think the option to navigate into one of the tabs using a keyword, while still keeping them visually as tabs is the best compromise for user friendliness for new users and quick navigation for experienced users. Also putting the community scripts into their own tab would much help to de-clutter the main menu. Having both options has no disadvantages as the tabs don't take up much space and provide great value in terms of categorization. 

Personally, I found the tabs very helpful when I first started using Kit and much preferred having the Community scripts separated into their own tab. I would very much appreciate if you would give us the option to restore the tabs, but also consider turning them on by default for new users. Thank you.

**Reply by johnlindquist** [reply]
- I've added an `.env` option of `KIT_MAIN_V1=true` to re-enable the old main menu UI. It will be available in the next build
- I've moved the community scripts under a keyword/script so they don't show in the main results

**Comment by patrikzudel** [comment]
> * I've added an `.env` option of `KIT_MAIN_V1=true` to re-enable the old main menu UI. It will be available in the next build
> 
>     * I've moved the community scripts under a keyword/script so they don't show in the main results

Has this ever been implemented? I've had `KIT_MAIN_V1=true` in my `.env` for a while and it isn't working :/

**Reply by johnlindquist** [reply]
@patrikzudel Apologies, I renamed it to:

```
KIT_MAIN_SCRIPT=v1
```
